perun-core/src/main/java/cz/metacentrum/perun/core/entry/AttributesManagerEntry.java

```diff
@@ -1327,6 +1324,8 @@ public void deleteAttribute(PerunSession sess, AttributeDefinition attributeDefi
 		getPerunBl().getMembersManagerBl().checkMemberExists(sess, member);
 		List<Attribute> attributes = getAttributesManagerBl().getRequiredAttributes(sess, resource, member, workWithUserAttributes);
 		Iterator<Attribute> attrIter = attributes.iterator();
+		User user = getPerunBl().getUsersManagerBl().getUserByMember(sess, member);
+		Facility facility = getPerunBl().getResourcesManagerBl().getFacility(sess, resource);
```

Comment:
This is not optimal. You get these data even when you don't need them (e.g. member-resource attributes)